src/webui/webapplication.cpp

```diff
@@ -67,7 +68,6 @@
 #include "api/torrentcreatorcontroller.h"
 #include "api/torrentscontroller.h"
 #include "api/transfercontroller.h"
-#include "freediskspacechecker.h"
```

Comment:
It seems `QTimer` header can also be removed.

Reply:
> It seems `QTimer` header can also be removed.

You're right. Done.